lifecycle/support/information-policy.md

```diff
@@ -1,5 +1,91 @@
+  - Development teams must utilize separate development or staging databases for testing and development purposes.
+  - Production databases should only be accessed for specific security patching or debugging purposes where data is likely to be the cause.
+  - When accessing production databases for these purposes, every effort should be made to avoid accessing or exposing sensitive data.
+  - Once the debugging or security patching is complete and a fix is in place, any data accessed from the production database must be immediately removed from the developer's local computer.
+
+- Access to GitHub:
+  - Access to GitHub repositories is granted on a need-to-know basis.
+  - Only team members who require access for their specific roles and responsibilities are granted permission to GitHub repositories.
+  - Access to GitHub repositories must be requested and approved by the Head of Engineering or the Lead Support Engineer.
+  - Team members granted access to GitHub repositories are responsible for adhering to the organization's code of conduct and security practices when interacting with the repositories.
+
+- Protection of Confidential Data:
+  - Under no circumstances should confidential data be held in a publicly accessible environment.
+  - Confidential data should be stored in secure, restricted-access systems and environments, following the organization's data protection guidelines.
+  - Data transmission involving confidential information should be encrypted using approved encryption protocols and methods.
+  - Any accidental exposure or unauthorized disclosure of confidential data must be reported immediately to the designated security officer or incident response team for appropriate action.
+
+- Secure Coding Practices:
+  - See our secure coding policy here.
+
+- Data Privacy and Consent:
+  - Respect user privacy and obtain appropriate consent for collecting, processing, and storing personal data in compliance with applicable privacy laws and regulations.
+  - Minimize the collection and storage of personally identifiable information (PII) to only what is necessary for the intended purpose.
+
+- Secure Development Environment:
+  - See our secure development environment policy here.
+
+- Test Data Management:
+  - Use anonymized or synthetic test data instead of real production data whenever possible to minimize the exposure of sensitive information during development and testing.
+  - Ensure that test data containing sensitive or personally identifiable information is securely stored, protected, and removed after its intended use.
+
+- Third-Party Libraries and Dependencies
+  - Vet and assess the security and privacy practices of third-party libraries, frameworks, and dependencies before integrating them into the codebase.
+  - Regularly update and patch third-party components to address known security vulnerabilities.
+ 
+
+
```

Comment:
I just want to make sure there's nothing missing at the end of this doc as there are many blank lines after